tests/test_enterprise/api/test_views.py

```diff
@@ -219,11 +191,11 @@ def test_user_view(self, factory, url, sorting_key, model_items, expected_json):
             reverse('enterprise-customer-entitlement-list'),
             itemgetter('enterprise_customer'),
             [{
-                'enterprise_customer__uuid': 'd1098bfb-2c78-44f1-9eb2-b94475356a3f',
+                'enterprise_customer__uuid': fake_catalog_api.FAKE_UUIDS[0],
                 'entitlement_id': 1
             }],
             [{
-                'enterprise_customer': 'd1098bfb-2c78-44f1-9eb2-b94475356a3f',
+                'enterprise_customer': fake_catalog_api.FAKE_UUIDS[0],
```

Comment:
- It does read weird, and for that reason I think we should just import it directly rather than through `fake_catalog_api`. Besides, we just care that they're fake UUIDs, not where they came from, so that should be fine. With that said, I'm going to move `FAKE_UUIDS` to `test_utils/__init__.py` instead -- makes more sense to be there.
- It's not about being more DRY, it's that it's easier to manage which UUIDs you're using where. It's easier to see the difference between `FAKE_UUIDS[0]` and `FAKE_UUIDS[1]` vs. `d1098bfb-2c78-44f1-9eb2-b94475356a3f` and `d3098bfb-2c78-44f1-9eb2-b94475356a3f`. This is important when conflicts matter.
- Using different UUIDs is all about preventing conflict -- they're *supposed* to be, well, universally unique :P

What I'll do is move `FAKE_UUIDS` so they're portable from `test_utils` directly, thus handling the readability concern, and I think changing things back to `DEFAULT_ENTERPRISE_UUID` will cause an inconsistency that'll cause detecting conflicts difficult when writing new tests that require multiple UUIDs, including `DEFAULT_ENTERPRISE_UUID`.